src/app/pages/psbt-request/components/psbt-decoded-request/psbt-input-output-pair/psbt-inscription.tsx

```diff
@@ -0,0 +1,12 @@
+import { InscriptionPreview } from '@app/components/inscription-preview-card/components/inscription-preview';
+import { useInscription } from '@app/query/bitcoin/ordinals/inscription.hooks';
+
+export function PsbtInscription(props: { path: string }) {
+  const { path } = props;
+  const { isLoading, isError, data: inscription } = useInscription(path);
+
+  if (isLoading) return null;
```

Comment:
`isLoading || isError` ?

Reply:
Yeah, was separating to possibly handle differently, but thanks for reminding me.